launch_ros/examples/lifecycle_pub_sub_launch.py

```diff
@@ -36,7 +36,7 @@ def main(argv=sys.argv[1:]):
 
     # Prepare the talker node.
     talker_node = launch_ros.actions.LifecycleNode(
-        node_name='talker',
+        name='talker', namespace='',
```

Comment:
The current change is ok, but this other version should still be working:
```suggestion
        name='talker'
```

Reply:
7ddcc1a